polaris-tokens/src/token-groups/color.ts

```diff
@@ -148,22 +159,22 @@ export type ColorTextAlias =
   | 'text-warning-strong';
 
 type ColorExperimentalAlias =
-  | 'subdued-link'
-  | 'bg-input-hover'
-  | 'bg-input-active'
-  | 'bg-transparent'
-  | 'bg-transparent-subdued'
-  | 'bg-transparent-hover'
-  | 'bg-transparent-active'
-  | 'bg-inverse-transparent-hover'
-  | 'bg-inverse-transparent-active';
+  | 'experimental-subdued-link'
```

Comment:
I didn't extract this in the `ColorTextAlias` because i'm not sure if its a text token

Reply:
I think this was added as more of a test to make sure the new tokens were being generated properly. If it's not used we can remove it for now, rename `ColorExperimentalAlias` to `ColorExperimentalBackgroundAlias` and remove the extract above which I think would be cleaner. We'll add another group for experimental text colors when they're ready  in figma